src/components/pages/WelcomePage.tsx

```diff
@@ -73,6 +74,15 @@ interface PostItem {
   date?: moment.Moment;
 }
 
+interface Place {
```

Comment:
Add docs and if this interface is needed anywhere else in the project move it to own .ts file and export it from there. This is the way it is done in other projects.